boba/cli.py

```diff
@@ -48,14 +52,46 @@ def print_help(err=''):
     ctx.exit()
 
 
+def run_batch_of_universes(folder, universes):
```

Comment:
It will be best to move all code related to `boba run` to a separate file! This `cli.py` is supposed to be a simple file mapping CLI commands to corresponding functions.